openedx_ledger/views.py

```diff
@@ -2,5 +2,96 @@
 Views for the openedx_ledger app.
 """
 # from django.shortcuts import render
+from django.http import HttpResponseRedirect
+from django.shortcuts import render
+from django.urls import reverse
+from django.views.generic import View
 
-# Create your views here.
+from openedx_ledger.api import reverse_full_transaction
+from openedx_ledger.models import Transaction
+from openedx_ledger.signals.signals import ADMIN_INVOKED_TRANSACTION_UNENROLL
+
+
+# TODO: Should we add a base class to do auth checks?
+class UnenrollLearnersView(View):
+    """
+    Admin view for the canceling platform enrollments form.
+    """
+    template = "edx_ledger/admin/unenroll.html"
+
+    def get(self, request, transaction_id):
+        """
+        Handle GET request - render "Cancel transaction without refund" form.
+
+        Arguments:
+            request (django.http.request.HttpRequest): Request instance
+            transaction_uuid (str): Enterprise Customer UUID
+
+        Returns:
+            django.http.response.HttpResponse: HttpResponse
+        """
+        return render(
+            request,
+            self.template,
+            {'transaction': Transaction.objects.get(uuid=transaction_id)}
+        )
+
+    def post(self, request, transaction_id):
+        """
+        Handle POST request - handle form submissions.
+
+        Arguments:
+            request (django.http.request.HttpRequest): Request instance
+        """
+        print("sending admin invoked transaction unenroll signal")
```

Comment:
nit: should the print statement be removed?

Reply:
it certainly should, I'll swap it over to a log